<problems>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>235</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyArgumentListInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Incorrect call arguments</problem_class>
  <description>Unexpected argument</description>
  <highlighted_element>question=self.question</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>22</length>
</problem>
</problems>